ios/Surveys/Resources/Localizations/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,3 @@
+"authentication.fields.email" = "Email";
+"authentication.fields.password" = "Password";
+"authentication.fields.login" = "Log in";
```

Comment:
should this be button?
```suggestion
"authentication.button.login" = "Log in";
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/81/commits/9e06a002678aacd2bd2b219835f3f7c5a070f1f6